packages/prosess-tilkjent-ytelse/src/components/TilkjentYtelse.jsx

```diff
@@ -35,26 +35,46 @@ const getOptions = nyePerioder => {
   };
 };
 
-const createTooltipContent = (intl, item) => `
+const createTooltipContent = (intl, item, getKodeverknavn) => {
+  const { formatMessage } = intl;
+  const periodeDato = `${moment(item.fom).format(DDMMYY_DATE_FORMAT)} - ${moment(item.tom).format(DDMMYY_DATE_FORMAT)}`;
+  return `
   <p>
-    ${moment(item.fom).format(DDMMYY_DATE_FORMAT)} - ${moment(item.tom).format(DDMMYY_DATE_FORMAT)}
-    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-     ${intl.formatMessage(
+    ${periodeDato}
+     ${formatMessage(
        { id: calcDaysAndWeeksWithWeekends(moment(item.fom), moment(item.tom)).id },
        {
          weeks: calcDaysAndWeeksWithWeekends(moment(item.fom), moment(item.tom)).weeks,
          days: calcDaysAndWeeksWithWeekends(moment(item.fom), moment(item.tom)).days,
        },
      )}
-    <br />&nbsp;
-    ${intl.formatMessage(
+    <br />
+    ${formatMessage(
       { id: 'Timeline.tooltip.dagsats' },
       {
         dagsats: item.dagsats,
       },
     )}
+    <br />
+    ${
+      (item.andeler || []).length > 1
+        ? item.andeler
+            .map(andel =>
+              formatMessage(
+                { id: 'Timeline.tooltip.dagsatsPerAndel' },
+                {
+                  arbeidsgiver: createVisningsnavnForAndel(andel, getKodeverknavn),
+                  dagsatsPerAndel: Number(andel.refusjon) + Number(andel.tilSoker),
+                  br: '<br />',
+                },
+              ),
+            )
+            .join('')
```

Comment:
@erlendev , har du en bedre løsningen på denne. Den må joines da komboen formatMessage inne i en map inne i en literal blir en array. Har prøvd flere ting som å bytte ut med <FormattedMessage /> osv.

Reply:
Denne skjønte jeg ikke helt. Hva er det du prøver å løse her? Hvis målet er å slå sammen resultatstrengene fra .map()-kallet kan jeg ikke se noe i veien for å bruke join :)

Reply:
Ja, det er det.